src/software/jetson_nano/ansible/playbooks/remote_flash.yml

```diff
@@ -0,0 +1,77 @@
+---
+
+- name: Remote flashing
+  hosts: THUNDERBOTS_HOSTS
+  vars:
+   all_services:
+     - thunderloop
+     - display
+     - announcement
+     - redis
```

Comment:
you can remove redis here, `sudo apt-get install redis` sets it up already

Reply:
removed redis from all files